astropy/units/format/fits.py

```diff
@@ -107,7 +106,7 @@ def _get_unit_name(cls, unit):
         return name
 
     @classmethod
-    def to_string(cls, unit):
+    def to_string(cls, unit, inline=True):
```

Comment:
Wait, isn't this API change? Are you sure you don't need change log?

Reply:
This is captured by #14393, but will be updated in #14449,,,